JavaScript/src/js/pokemon.js

```diff
@@ -0,0 +1,94 @@
+class Pokemon{
+    constructor(name, tipo, attack,defense){
+        this._name = name;
+        this._tipo = tipo;
+        this._attack = attack;
+        this._defense = defense;
+    }
+
+    get name() {
+        return this._name;
+    }
+    
+    get tipo() {
+        return this._tipo;
+    }
+
+    get attack(){
+        return this._attack;
+    }
+
+    get defense(){
+        return this._defense;
+    }
+}
+
+function defineEffectiveness(typeFirstPokemon, typeSecondPokemon){
+    valueEffentiveness = 0;
+    switch (typeFirstPokemon){
```

Comment:
on js try to avoid "switch" statement, read these articles as reference:

- http://www.nikola-breznjak.com/blog/javascript/converting-a-javascript-switch-statement-into-a-function-lookup/
- https://toddmotto.com/deprecating-the-switch-statement-for-object-literals/